src/app/event-external-root/event-external-root.controller.js

```diff
@@ -3,14 +3,34 @@
 angular.module('esn.calendar.event-consultation')
   .controller('CalEventExternalRootController', CalEventExternalRootController);
 
-function CalEventExternalRootController($scope, result) {
+function CalEventExternalRootController($rootScope, $location, result, calEventExternalRootService) {
   const self = this;
 
+  $rootScope.$on('UPDATE_ACTION_EXCAL', function(event, newLink) {
+    const newJWT = parseJwtKey(newLink);
```

Comment:
If UPDATE_ACTION_EXCAL is only used to update the JWT, we can emit only the JWT if present in the link.

Reply:
we need to parse jwt key anyway (before emit or after)